### PR TITLE
Removing enterprise label from cloud deployment pages

### PR DESCRIPTION
**Pull request by lidiazuin**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates.

You can view the updated docs at https://neo4j-docs-operations-1270.surge.sh

**Comment by renetapopova** [comment]
Thanks, @lidiazuin. Have you spoken with @edrandall-dev about this as stated in the [Trello card](https://trello.com/c/94rWw6kv/185-remove-the-enterprise-edition-label-from-cloud-deployments)? What does he think about it? This card is not only about removing the labels but also rewording the content as of now it only focuses on the Enterprise edition.

**Comment by lidiazuin** [comment]
> Thanks, @lidiazuin. Have you spoken with @edrandall-dev about this as stated in the [Trello card](https://trello.com/c/94rWw6kv/185-remove-the-enterprise-edition-label-from-cloud-deployments)? What does he think about it? This card is not only about removing the labels but also rewording the content as of now these pages only focus on the Enterprise edition.

Yes, I just checked with him and these changes only concern to self-managed cloud deployments, not Aura, so our suggestion here in the PR is correct. The only mention to Aura should be the one already existing in the Cloud deployments index page. Currently the links only take to product pages, but we could add a link to the Aura docs too, if you think it's relevant.

File: modules/ROOT/pages/cloud-deployments/neo4j-aws.adoc [hunk 1]
```diff
@@ -131,15 +131,6 @@ After the installation finishes successfully, the CloudFormation template provid
 When the CloudFormation template creates a new Neo4j cluster, an Auto Scaling group (ASG) is created and tagged with the minor version of the installed Neo4j database.
 If you add more EC2 instances to your ASG, they will be installed with the same minor version, ensuring that all Neo4j cluster servers are installed with the same version, regardless of when the EC2 instances were created.
 
-== Licensing
```
Comment (renetapopova) [inline modules/ROOT/pages/cloud-deployments/neo4j-aws.adoc:135]: This is still valid for Enterprise. So, maybe you can add a label here.

File: modules/ROOT/pages/cloud-deployments/neo4j-azure.adoc [hunk 2]
```diff
@@ -125,15 +125,6 @@ If the validation fails, it might be because you have chosen VMs that are too la
 When the ARM template creates a new Neo4j cluster, a Virtual Machine Scale Set (VMSS) is created and tagged with the minor version of the installed Neo4j database.
 If you add more VM instances to your VMSS, they will be installed using the same minor version, ensuring that all Neo4j cluster servers are installed with the same version, regardless of when the VM instances were created.
 
-== Licensing
```
Comment (renetapopova) [inline modules/ROOT/pages/cloud-deployments/neo4j-azure.adoc:129]: same here

File: modules/ROOT/pages/cloud-deployments/neo4j-gcp.adoc [hunk 3]
```diff
@@ -118,15 +118,6 @@ After the installation finishes successfully, the Deployment Manager template pr
 The Neo4j Browser can be easily launched in a new window by clicking the button entitled *Log into the Neo4j Browser*.
 ====
 
-== Licensing
```
Comment (renetapopova) [inline modules/ROOT/pages/cloud-deployments/neo4j-gcp.adoc:122]: and here

**Review by renetapopova — APPROVED** [review]
Looks good, thanks, @lidiazuin. I just added a comment about the licensing sections, stating that they are still valid for EE and we should keep them with a label.

**Pull request merged**